src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -46,4 +46,9 @@ export async function getCached(
   return instance
 }
 
+export function clearCache(): null {
+  instance = null
+  return instance
```

Comment:
Question: Is it necessary for this function to return something or should it rather be `void`? I am asking as I am working on the chain connection to Portablegabi and my code is very similar except for `clearCache` having no return value in my case.

Reply:
Yup makes sense, I'm removing the return value since indeed `clear`-like methods e.g. in web APIs don't return.